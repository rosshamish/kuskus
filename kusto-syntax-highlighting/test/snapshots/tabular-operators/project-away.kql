// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/projectawayoperator

print a2='a2', b = 'b', a3='a3', a1='a1'
| project-away a*
