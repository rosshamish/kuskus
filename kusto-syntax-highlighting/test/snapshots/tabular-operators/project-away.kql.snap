>// https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/projectawayoperator
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.kusto comment.line.kusto
>
>print a2='a2', b = 'b', a3='a3', a1='a1'
#^^^^^ source.kusto keyword.other.query.kusto
#     ^^^^ source.kusto
#         ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
#          ^^ source.kusto string.quoted.single.kusto
#            ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
#             ^^^^^^ source.kusto
#                   ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
#                    ^ source.kusto string.quoted.single.kusto
#                     ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
#                      ^^^^^ source.kusto
#                           ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
#                            ^^ source.kusto string.quoted.single.kusto
#                              ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
#                               ^^^^^ source.kusto
#                                    ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
#                                     ^^ source.kusto string.quoted.single.kusto
#                                       ^ source.kusto string.quoted.single.kusto punctuation.definition.string.kusto
>| project-away a*
#^^ source.kusto
#  ^^^^^^^^^^^^ source.kusto keyword.other.query.kusto
#              ^^^^ source.kusto
>